optima/calibration.py

```diff
@@ -181,7 +186,7 @@ def convert(pars, parlist, parvec=None):
     
 
 
-    def errorcalc(parvec=None, pars=None, parlist=None, project=None, bestindex=0):
+    def errorcalc(parvec=None, pars=None, parlist=None, project=None, fitto='prev', bestindex=0, doplot=False, verbose=2):
```

Comment:
I think we need to rename this. There are many, many ways to quantify goodness of fit by some kind of "error calc". What you've got here looks like it's calculating the MAD/mean ratio or weighted absolute percent error. We could call this madmeanratiocalc or wapecalc, and then down the track we could consider adding other options (MAE, SMAPE, good old MSE).